acceptance/acceptance_test.go

```diff
@@ -96,6 +96,13 @@ func TestInprocessMode(t *testing.T) {
 }
 
 func testAccept(t *testing.T, inprocessMode bool, singleTest string) int {
+	parallelFlag := flag.Lookup("test.parallel")
+	if parallelFlag != nil {
+		t.Logf("GOMAXPROCS=%d, parallel=%s, parallel_default=%s", runtime.GOMAXPROCS(0), parallelFlag.Value.String(), parallelFlag.DefValue)
+	} else {
+		t.Logf("GOMAXPROCS=%d", runtime.GOMAXPROCS(0))
```

Comment:
Do we need to log this? It's safe to assume this is always > 1.

Reply:
dropped this.